ResourceManager/GitExtensionsControl.cs

```diff
@@ -74,7 +74,12 @@ public virtual void TranslateItems(ITranslation translation)
 
         #region Hotkeys
 
-        /// <summary>Checks if a hotkey wants to handle the key and execute it.</summary>
+        /// <summary>
+        /// Checks if a hotkey wants to handle the key and execute that hotkey
+        /// (without propagating an unhandled key to the base class function <cref>ProcessCmdKey</cref>).
+        /// Can be overridden in order to execute a hotkey of other (visible) subcontrols
+        /// if this focused/queried control does not have such an (active) hotkey itself.
```

Comment:
I do not understand the last line.

Reply:
> I do not understand the last line.

This function `ProcessHotkey` is called on key press by the function `Control.ProcessCmdKey` (if the control is *focused*)
or when one of its parent controls asks / *queries* whether this control has an (active) hotkey.
(If a hotkey is not active, the key is treated as "not handled yet" and is propagated to the other controls and upstream.)

How would you reword the comment?

Reply:
```suggestion
        /// Checks if the control wants to handle the key and execute that hotkey
        /// (without propagating an unhandled key to the base class function as in <cref>ProcessCmdKey</cref>).
        /// Can be overridden in order to execute a hotkey for a (visible) subcontrol
        /// (if this focused/queried control does not have such an (active) hotkey itself).
```

Reply:
Thank you, done.